### PR TITLE
Issue all certificates [WIP]

### LINKED ISSUE #757: [feature] Allow to generate all certificates for one course in one click
in courses with many participants can be tedious to generate the certificates one at a time.

### PR DESCRIPTION
**Pull request by sonlinux**
fix #757

### CODE REVIEW

File: django_project/certification/views/certificate.py [hunk 1]
```diff
@@ -861,3 +862,94 @@ def preview_certificate(request, **kwargs):
...
+
+        current_site = request.META['HTTP_HOST']
+
+        # for every attendee, generate a certificate and issue it.
+        for attendee in attendees:
+            certificate = DummyCertificate(course, attendee, project)
+
+            generate_pdf(pathname, project, course, attendee, certificate,
+                         current_site)
+        messages.success(request, '{} certificates have been issued'.format(len(attendees)),
```
Comment (ismailsunni) [inline django_project/certification/views/certificate.py]: __[Flake8]__

[E501] line too long (92 > 79 characters)

File: django_project/certification/views/certificate.py [hunk 2]
```diff
@@ -861,3 +862,94 @@ def preview_certificate(request, **kwargs):
...
+
+    url = reverse('course-detail', kwargs={
+        'project_slug': project_slug,
+        'organisation_slug': organisation_slug,
+        'slug': course_slug
+    })
+
+    project_folder = (project.name.lower()).replace(' ', '_')
+    if request.method == 'POST':
+        site = request.get_host()
```
Comment (ismailsunni) [inline django_project/certification/views/certificate.py]: __[Flake8]__

[F841] local variable 'site' is assigned to but never used

**Comment by timlinux** [comment]
@sonlinux 14 days this PR is languishing with failing tests - please surprise and delight me with ready to merge PR's

**Comment by timlinux** [comment]
23 days @sonlinux

**Pull request closed without merging**